EnglishFilter/sections/antiadblock.txt

```diff
@@ -449,7 +449,7 @@ pornissimo.org,pornobit.me,porntn.com,femdomtb.com,pornoham.me,tubsexer.com,home
 !
 ! ai_front anti-adblock
 ! #%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
-ubuntudde.com,mettablog.com,secondlifetranslations.com,charbelnemnom.com,lowkeytech.com,pinayvids.com,gcertificationcourse.com,game-owl.com,plugincrack.com,manyakan.com#%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
+jootc.com,ubuntudde.com,mettablog.com,secondlifetranslations.com,charbelnemnom.com,lowkeytech.com,pinayvids.com,gcertificationcourse.com,game-owl.com,plugincrack.com,manyakan.com#%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
```

Comment:
That's ok, but you forgot about content blockers. I will add it, please take a look.